package.json

```diff
@@ -36,32 +36,32 @@
     "@bam.tech/react-native-image-resizer": "3.0.11",
     "@comapeo/core-react": "7.2.0",
     "@comapeo/ipc": "6.0.1",
-    "@expo-google-fonts/rubik": "0.4.1",
-    "@formatjs/intl-getcanonicallocales": "2.5.5",
-    "@formatjs/intl-locale": "4.2.11",
-    "@formatjs/intl-pluralrules": "5.4.4",
-    "@formatjs/intl-relativetimeformat": "11.4.11",
-    "@gorhom/bottom-sheet": "5.2.4",
+    "@expo-google-fonts/rubik": "0.4.2",
```

Comment:
This namespacing was a little confusing to me. I'm still not sure if `@expo`(as opposed to `expo`- with no "a") is managed by the expo upgrades.